comms/selection_test.go

```diff
@@ -0,0 +1,126 @@
+package comms
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type thing struct {
+	name   string
+	rating int
+}
+
+func (t thing) String() string {
+	return fmt.Sprintf("%s (+%d)", t.name, t.rating)
```

Comment:
Oh this is interesting. `Display` just uses the `String` method, whatever it is, to format. Clever. Little golang :bulb: moment.